tests/transports-integrations/run_integration_tests.py

```
@@ -0,0 +1,217 @@
+
+    if args.check_keys:
+        print("🔑 API Key Status:")
+        for integration in available_integrations:
+            print(f"  ✅ {integration.upper()}: Available")
+        for integration in missing_integrations:
+            print(f"  ❌ {integration.upper()}: Missing")
+        return
+
+    if args.show_models:
+        # Import and show model configuration
+        from tests.utils.models import print_model_summary
```

Pylint (3.3.7)

[convention] Import outside toplevel (tests.utils.models.print_model_summary)

(C0415)